test/core/componentTests.ts

```diff
@@ -203,13 +203,13 @@ describe("Component behavior", () => {
   it("clipPath works as expected", () => {
     assert.isFalse(c.clipPathEnabled, "clipPathEnabled defaults to false");
     c.clipPathEnabled = true;
-    var expectedClipPathID = c.getID();
     c.anchor(svg);
     c.computeLayout({ x: 0, y: 0 }, 100, 100);
     c.render();
+    var expectedClipPathID = (<any>c)._boxContainer[0][0].firstChild.id;
```

Comment:
Since this was peeled directly off of the DOM, it's not really "expected" anymore -- it's the actual value.

Selecting this will be more robust if you do
```Typescript
(<any> c)._boxContainer.select("clipPath").attr("id");
```